openwisp_radius/templates/radius_accounting_start.html

```diff
@@ -0,0 +1,9 @@
+{% load i18n %}{% load l10n %}
+<div class="msg">
+<p>
+{% trans "New session has been started for your account with username:" %} {{ user.get_username }}
```

Comment:
try to use:

``` 
<p>.....</p>
<p>......</p>
```
To help strip_tags to do its job without leaving too many whitespace.